appium/tests/screenobjects/new-message.screen.ts

```diff
@@ -112,6 +122,17 @@ class NewMessageScreen extends BaseScreen {
     expect(value).toEqual(`  ${recipient}  `);
   };
 
+  attachmentName = async (name: string) => {
+    const selector = `-ios class chain:**/XCUIElementTypeStaticText[\`label == "${name}"\`]`;
```

Comment:
I would add incremental accessibility ids to attachments. So first attachment would be `attachment1` then `attachment-1` and so on (or you can start with 1 if you want)

Then you would be getting attachment-0 in tests by selector, and make sure it has the right content.

I think that's the most prudent way to test.

Reply:
Yes, it should be a good solution for quicker search of elements 👍